reduction/lr_reduction/reduction_template_reader.py

```diff
@@ -76,6 +78,7 @@ def __init__(self):
         self.paralyzable: bool = True
         self.dead_time_value = 4.2
         self.dead_time_tof_step = 100
+        self.dead_time_threshold_ratio: Optional[Float] = Property.EMPTY_DBL
```

Comment:
Since this parameter will always be written as part of the template, we need to think about how to make it optional, i.e. be able to turn "off" the deadtime threshold. Perhaps if the threshold is zero, then it is "off"? Since valid values are > 1 per our slack discussion.
When the value for `EMPTY_DBL` is written to the template file and read back, it is likely that Mantid won't recognize it as `EMPTY_DBL` anymore.